build failed:
```
======================================================================
ERROR: test_create_default_from_export_item (corehq.apps.export.tests.test_export_item.TestExportItemGeneration)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/export/tests/test_export_item.py", line 19, in test_create_default_from_export_item
    column = ExportColumn.create_default_from_export_item([None], self.item, 3)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/export/models/new.py", line 87, in create_default_from_export_item
    for app_id, version in build_ids_and_versions.iteritems():
AttributeError: 'int' object has no attribute 'iteritems'
```

--- a/corehq/apps/app_manager/dbaccessors.py
+++ b/corehq/apps/app_manager/dbaccessors.py
@@ -178,8 +178,11 @@ def get_built_app_ids_for_app_id(domain, app_id, version=None):
 
 def get_latest_built_app_ids_and_versions(domain, app_id=None):
     """
-    Returns all the latest build app_ids and version in a dictionary:
-    {app_id: latest_version}
+    Returns all the latest app_ids and versions in a dictionary.
+    :param domain: The domain to get the app from
+    :param app_id: The app_id to get the latest version from. If not specified gets latest versions of all
+        apps in the domain
+    :returns: {app_id: latest_version}
     """
     from .models import Application
     key = [domain, app_id] if app_id else [domain]

--- a/corehq/apps/export/models/new.py
+++ b/corehq/apps/export/models/new.py
@@ -28,8 +28,13 @@ from corehq.apps.export.const import (
 class ExportItem(DocumentSchema):
     """
     An item for export.
-    path is a question path like ["my_group", "q1"] or a case property name
-    like ["date_of_birth"].
+
+    path: A question path like ["my_group", "q1"] or a case property name
+        like ["date_of_birth"].
+
+    label: The label of the corresponding form question, or the case property name
+    tag: Denotes whether the property is a system, meta, etc
+    last_occurrences: A dictionary that maps an app_id to the last version the export item was present
     """
     path = ListProperty()
     label = StringProperty()

--- a/corehq/apps/export/tests/test_export_item.py
+++ b/corehq/apps/export/tests/test_export_item.py
@@ -7,30 +7,31 @@ from corehq.apps.export.models import (
 
 
 class TestExportItemGeneration(SimpleTestCase):
+    app_id = '1234'
 
     def setUp(self):
         self.item = ExportItem(
             path=['data', 'question1'],
             label='Question One',
-            last_occurrence=3,
+            last_occurrences={self.app_id: 3},
         )
 
     def test_create_default_from_export_item(self):
-        column = ExportColumn.create_default_from_export_item([None], self.item, 3)
+        column = ExportColumn.create_default_from_export_item([None], self.item, {self.app_id: 3})
 
         self.assertEqual(column.show, True)
         self.assertEqual(column.label, 'Question One')
         self.assertEqual(column.selected, True)
 
     def test_create_default_from_export_item_deleted(self):
-        column = ExportColumn.create_default_from_export_item([None], self.item, 4)
+        column = ExportColumn.create_default_from_export_item([None], self.item, {self.app_id: 4})
 
         self.assertEqual(column.show, False)
         self.assertEqual(column.label, 'Question One')
         self.assertEqual(column.selected, False)
 
     def test_create_default_from_export_item_not_main_table(self):
-        column = ExportColumn.create_default_from_export_item(['other_table'], self.item, 3)
+        column = ExportColumn.create_default_from_export_item(['other_table'], self.item, {self.app_id: 3})
 
         self.assertEqual(column.show, True)
         self.assertEqual(column.label, 'Question One')

